components/calendar/day.js

```diff
@@ -1,18 +1,15 @@
 import Event from './event';
+import moment from 'moment';
 
-const DAY_NAMES = [  'Søndag', 'Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag' ];
-const MONTH_NAMES = [ 'januar', 'februar', 'mars', 'april', 'mai', 'juni', 'juli', 'august', 'september', 'oktober', 'november', 'desember' ];
+moment.locale('nb')
```

Comment:
Mind elaborating on this? I'm importing moment in every component it's used, importing it only in in the parent doesn't seem to work from my testing. I will move that line in particular to the index of the Calendar component however. Setting the locale once there works, so there's no point in setting it to the same value for every Day (of which we have about 14 of typically),